the build system doesn't check for this and uses the fallback, but those headers
are broken with musl and kernel include order stuff
--
--- a/snapper/BtrfsUtils.cc
+++ b/snapper/BtrfsUtils.cc
@@ -31,7 +31,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <unistd.h>
-#ifdef HAVE_LIBBTRFS
+#if 1
 #include <btrfs/ioctl.h>
 #include <btrfs/send-utils.h>
 #else
